### PR TITLE
Race2 auv

### PR DESCRIPTION
**Pull request by farhangnaderi**
@mzhouURI Please check if it is ok. I tested on my side!

image

### CODE REVIEW

File: launch/simulation.launch [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     <arg name="scenario" default="ocean"/>
 
-    <arg name="robot_name" default="race2"/>
+    <arg name="robot_name" default="changename"/>
 
     <arg name="data_path" default="$(find world_of_stonefish)/data"/>
```
Comment (linzha0) [inline launch/simulation.launch:21]: HI, this looks good

**Comment by farhangnaderi** [comment]
@mzhouURI I squashed all commits! Must be correct now! You may do a final check!

File: world/ocean_race2.scn.xacro [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+    This file is part of ALPHA AUV project.
+
+    Author: Emir Cem Gezer
```
Comment (mzhouURI) [inline world/ocean_race2.scn.xacro:5]: Author needs to the changed

File: world/ocean_race2.scn.xacro [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    <xacro:arg name="robot_name" default="changeme"/>
+    <xacro:include filename="$(find world_of_stonefish)/vehicles/$(arg robot_name).scn.xacro"/>
+
+    <xacro:race2 robot_name="$(arg robot_name)" xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
+
+    <environment>
+        <ned latitude="41" longitude="-71"/>
+        <ocean>
+            <water density="1031.0" jerlov="0.2"/>
```
Comment (mzhouURI) [inline world/ocean_race2.scn.xacro:21]: water density is not realistic

Comment (mzhouURI) [reply world/ocean_race2.scn.xacro:21]: Put 1025

File: world/ocean.scn.xacro
Comment (mzhouURI) [inline world/ocean.scn.xacro:1]: Is this just move the file back?
same for the simulation.launch

Comment (farhangnaderi) [reply world/ocean.scn.xacro:1]: Yes, I changed them back to their default.

**Pull request merged**